### ISSUE TITLE
Enhancement: Keep parameter list on one line

### ISSUE THREAD

**Issue by catesandrew** [jsdoc2md/dmd#29]
I have a couple minor enhancements to the [params list](https://github.com/jsdoc2md/dmd/blob/master/partials/all-docs/docs/body/params/params-list.hbs#L1-L6)
- Add a colon after `**Params**` followed by a blank line so the list that starts after it is rendered by a wider collection of applications.
- Keep the `name`, `type`, and `description` that follows on the same line. Simply add a `~` to the closing `if` that comes after `linked-type-list` so it becomes `{{~/if}}`.
- Skip rendering the `**Params**` all together if there are none in the list (not implemented here)

Before:
```hbs
{{#params}}**Params**
{{#each this~}}
{{indent}}- {{name}}{{#if type}} {{>linked-type-list types=type.names delimiter=" | " }}{{/if}}{{#unless (equal defaultvalue undefined)}} {{>defaultvalue equals=true ~}}{{/unless}}{{#if description}} - {{{inlineLinks description}}}{{/if}}
{{/each}}

{{/params~}}
```

After:
```hbs
{{#params}}**Params**:

{{#each this~}}
{{indent}}- {{name}} {{#if type~}}{{>linked-type-list types=type.names delimiter=" | " }} {{~/if}}{{#unless (equal defaultvalue undefined)}} {{>defaultvalue equals=true ~}}{{/unless}}{{#if description}} - {{{inlineLinks description}}}{{/if}}
{{/each}}

{{/params~}}
```

**Comment by 75lb** [comment]
points 1 and 3 fixed, thank you.. however, i didn't understand point 2 ("Keep the name, type, and description that follows on the same line") because they already are on the same line.. can you send a link to a broken instance please? 

you can see on [my testbed example](https://github.com/jsdoc2md/testbed/blob/master/build/example/options/dmd-options/param-list-format/3-dmd.md#jsdoc2md---param-list-format-list) that they are rendered on one line..

**Comment by 75lb** [comment]
points 1 and 3 released in dmd 1.3.5, please update

**Comment by catesandrew** [comment]
thanks, sorry for late reply, shit i got busy at work.
